.github/workflows/pr-helm-lint.yml

```diff
@@ -0,0 +1,54 @@
+# This workflow triggers helm linting on PRs and master branch pushes
+# that affect helm files
+name: Helm Chart Linting on Pull Requests and Master
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - helm/wfl/**
+  pull_request:
+    paths:
+      - helm/wfl/**
```

Comment:
I'm targeting helm/wfl specifically because we'd have to reconfigure this action a bit if we added other charts. Basic linting (just errors) will work on everything because chart-testing has logic to find charts based on ct.yaml, but the strict linting is just done via command line and uses a working directory of helm/wfl.